spec/std/http/client/response_spec.cr

```diff
@@ -169,14 +169,14 @@ class HTTP::Client
     end
 
     it "doesn't sets content length for 1xx, 204 or 304" do
-      [100, 101, 204, 304].each do |status|
+      [HTTP::Status::CONTINUE, HTTP::Status::SWITCHING_PROTOCOLS, HTTP::Status::NO_CONTENT, HTTP::Status::NOT_MODIFIED].each do |status|
```

Comment:
```suggestion
      response_codes = [:Continue, :SwitchingProtocols, :NoContent, :NotModified] of HTTP::Status
      response_codes.each do |status|
```

Reply:
CamelCased symbols look odd, they're almost explicitly used with snake_case.

Reply:
This isn't about the camel cased symbols, it's about using symbols in the first place.

Reply:
so let's use snake_case then.

Reply:
I don't see how that's related - here we're talking about symbols and the issue you mentioned is about enum members - 2 different things.